lib/mayaUsd/ufe/UsdMaterial.h

```diff
@@ -0,0 +1,50 @@
+// ===========================================================================
+// Copyright 2022 Autodesk, Inc. All rights reserved.
+//
+// Use of this software is subject to the terms of the Autodesk license
+// agreement provided at the time of installation or download, or which
+// otherwise accompanies this software in either electronic or hard copy form.
+// ===========================================================================
```

Comment:
Every file needs a license notice and copyright. Check any of the newer files for an example.